lesson17/src/js/main.js

```diff
@@ -0,0 +1,212 @@
+function showLoadingImage() {
+  const main = document.getElementById("main");
+  const fragment = document.createDocumentFragment();
+  const renderLoadingBox = document.createElement("div");
+  renderLoadingBox.classList.add("loading");
+  renderLoadingBox.id = "js-loading";
+  const loadingImage = document.createElement("img");
+  loadingImage.src = "images/loading.gif";
+  loadingImage.width = 80;
+  loadingImage.height = 80;
+  loadingImage.classList.add("loading__image");
+  fragment.appendChild(renderLoadingBox).appendChild(loadingImage);
+  document.body.insertBefore(fragment, main);
+}
```

Comment:
```suggestion
function showLoadingImage() {
  const main = document.getElementById("main");
  const renderLoadingBox = document.createElement("div");
  renderLoadingBox.classList.add("loading");
  renderLoadingBox.id = "js-loading";
  const loadingImage = document.createElement("img");
  loadingImage.src = "images/loading.gif";
  loadingImage.width = 80;
  loadingImage.height = 80;
  loadingImage.classList.add("loading__image");
  renderLoadingBox.appendChild(loadingImage);
  document.body.insertBefore(renderLoadingBox, main);
}
```
IMO & Q
I feel that the content is necessary though fragment is used in the part where it is not looped. How about writing it like this.
Is there a reason to use fragment?

Reply:
Certainly the fragment here is unnecessary!
Fixed it!

[c00e081](https://github.com/nyattsu72/hanson-frontend-cource/pull/25/commits/c00e081d91350d4237d05a25d40006ab1bb1225e)